src/dht/dht_settings.rs

```diff
@@ -101,7 +101,13 @@ impl DHTSettings {
     }
 }
 
-#[derive(Clone)]
+impl Default for DHTSettings {
+    fn default() -> Self {
+        Self::default()
```

Comment:
I think we should move the current implementation of `DHTSettings::default()` (lines 76-101) here, then delete `impl DHTSettings {...}` altogether.

```suggestion
        DHTSettings {
            token_secret_size: 10,
            max_peers_response: 128,
            max_sample_response: 50,
            min_sample_interval_secs: 10,
            router_ping_interval_secs: 900,
            reverify_interval_secs: 14 * 60,
            reverify_grace_period_secs: 15 * 60,
            verify_grace_period_secs: 60,
            get_peers_freshness_secs: 15 * 60,
            find_nodes_interval_secs: 33,
            find_nodes_skip_count: 32,
            max_torrents: 50,
            max_peers_per_torrent: 100,
            ping_check_interval_secs: 10,
            outgoing_request_prune_secs: 30,
            outgoing_reqiest_check_interval_secs: 30,
            read_only: false,
            routers: vec![
                "router.bittorrent.com:6881".to_string(),
                "router.utorrent.com:6881".to_string(),
                "dht.transmissionbt.com:6881".to_string(),
            ],
        }
```

Reply:
Done